tools/tests/router/test.js

```diff
@@ -49,7 +49,8 @@ function beforeUnload(callback, params) {
         confirmationDialog.removeChild(confirmButton);
         confirmationDialog.parentElement.removeChild(confirmationDialog);
         confirmButton.style.display = 'none';
-        callback.apply(...params);
+        // eslint-disable-next-line prefer-spread
+        callback.apply(null, params);

```

Comment:
because the linter wants the new syntax with the spread operator but when I do `callback(...params)` it failed. I realized why it is failing so I fixed this problem and removed disabling comment of the linter. You can review it in the latest commit